Samples/SampleApp.Tests/DomainTests.cs

```diff
@@ -15,7 +14,7 @@ public class DomainTests
         public DomainTests()
         {
             domainEntities = Types.InAssemblyOf<DomainClass>("Domain Entities", 
-                types => types.Where(t=>t.Namespace.StartsWith("SampleApp.Domain")));
+                type => type.Namespace.StartsWith("SampleApp.Domain"));
```

Comment:
Not really. Maybe you could return all implemented interfaces or base classes? Anyway, the method is still there, it's just obsolete. I could remove the `ObsoleteAttribute` and just let it sit there with the rest :smile: